services/worker/src/worker/job_runners/split/duckdb_index.py

```diff
@@ -112,16 +130,12 @@ def compute_index_rows(
         features = content_parquet_and_info["dataset_info"]["features"]
         column_names = ",".join('"' + column + '"' for column in list(features.keys()))
 
-        # look for string columns
-        string_columns = [
-            column
-            for column, feature in features.items()
-            if "dtype" in feature
-            and "_type" in feature
-            and feature["dtype"] == STRING_FEATURE_DTYPE
-            and feature["_type"] == VALUE_FEATURE_TYPE
-        ]
-        if not string_columns:
+        # look for indexable columns (= possibly nested columns containing string data)
+        # copy the features is needed but will be fixed with https://github.com/huggingface/datasets/pull/6189
```

Comment:
remove the comment, since https://github.com/huggingface/datasets/pull/6189 has been merged?

Reply:
will removed when updating `datasets` to a version that includes the fix

Reply:
oh right, it's in another repository, sorry